pom.xml

```diff
@@ -92,6 +92,28 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.2</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                        </manifest>
+                        <manifestEntries>
+                            <Build-Tag>${build-tag}</Build-Tag>
+                            <Git-Commit>${build-commit}</Git-Commit>
+                            <Git-Commit-Abbrev>${build-commit-abbrev}</Git-Commit-Abbrev>
+                            <Build-Version>${build-version}</Build-Version>
+                            <Build-Timestamp>${build-tstamp}</Build-Timestamp>
+                            <Source-Repository>${project.scm.connection}</Source-Repository>
```

Comment:
@marmoure The `project` properties you are referencing here need to be set. Take a look here:

1. You need an `<scm>` section. Example here - https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L33
2. You need a `<description>`. Example here - https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L16
3. You need a `<url>`. Example here -https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L17